Mathlib/Algebra/Category/AlgebraCat/Basic.lean

```diff
@@ -202,9 +214,9 @@ def toAlgEquiv {X Y : AlgebraCat R} (i : X ≅ Y) : X ≃ₐ[R] Y where
     change (i.inv ≫ i.hom) x = x
     simp only [inv_hom_id]
     rw [id_apply]
-  map_add' := by simp
-  map_mul' := by simp
-  commutes' := by simp
+  map_add' := i.hom.map_add -- Porting note: was `by simp`
+  map_mul' := i.hom.map_mul -- Porting note: was `by simp`
+  commutes' := i.hom.commutes -- Porting note: was `by simp`
```

Comment:
these `by simp` at the end used to be `by tidy`, I was playing around with alternatives and didn't leave a porting note, so we should probably change the porting note you added.

Reply:
Aah, good catch. Please go ahead!